.gitignore

```diff
@@ -89,3 +89,4 @@ fabric.properties
 
 # Android studio 3.1+ serialized cache file
 .idea/caches/build_file_checksums.ser
+*.json
```

Comment:
Is there a specific reason you added this

Reply:
oh no I dont know how this got there

Reply:
Please remove then